tests/utils/deployment/deployment.go

```diff
@@ -1,17 +1,69 @@
 package deployment
 
 import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/golang/glog"
+	"github.com/test-network-function/cnfcert-tests-verification/tests/globalhelper"
 	v1 "k8s.io/api/apps/v1"
 	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/utils/pointer"
+
+	. "github.com/onsi/gomega"
 )
 
+func isDeploymentReady(operatorNamespace string, deploymentName string) (bool, error) {
```

Comment:
In Utils we keep only define functions. Please move all non-define function to  ./tests/globalhelper/deployment.go